modules/core/src/main/java/org/apache/synapse/mediators/transform/XSLTMediator.java

```diff
@@ -330,6 +354,25 @@ private void performXSLT(MessageContext synCtx, SynapseLog synLog) {
 
             synLog.traceOrDebug("Transformation completed - processing result");
 
+            /**
+             * If user have set transform.xslt.result.disableBuild property to true, we do not build the message to
+             * OMElement,
+             */
+            if (targetPropertyName != null && resultBuilder instanceof StreamResultBuilder && synCtx.getProperty(
+                    TRANSFORM_XSLT_RESULT_DISABLE_BUILD) != null &&
```

Comment:
let's move synCtx.getProperty( TRANSFORM_XSLT_RESULT_DISABLE_BUILD) != null to same row

Reply:
Updated.